src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Administration/GeneralType.php

```diff
@@ -43,19 +45,52 @@ public function buildForm(FormBuilderInterface $builder, array $options)
-                'required' => true,
+                'label' => $this->trans('Automatically check for module updates', 'Admin.Advparameters.Feature'),
+                'help' => $this->trans('New modules and updates are displayed on the modules page.', 'Admin.Advparameters.Help'),
             ])
             ->add('check_ip_address', SwitchType::class, [
-                'required' => true,
+                'label' => $this->trans('Check the cookie\'s IP address', 'Admin.Advparameters.Feature'),
+                'help' => $this->trans('Check the IP address of the cookie in order to prevent your cookie from being stolen.', 'Admin.Advparameters.Help'),
             ])
             ->add('front_cookie_lifetime', TextType::class, [
-                'required' => true,
+                'label' => $this->trans('Lifetime of front office cookies', 'Admin.Advparameters.Feature'),
+                'help' => $this->trans('Set the amount of hours during which the front office cookies are valid. After that amount of time, the customer will have to log in again.', 'Admin.Advparameters.Help'),
+                'constraints' => [
+                    new Type(
+                        [
+                            'value' => 'numeric',
+                            'message' => $this->trans('The field is invalid', 'Admin.Notifications.Error'),
+                        ]
+                    ),
+                    new GreaterThanOrEqual(
+                        [
+                            'value' => 0,
+                            'message' => $this->trans('The field is invalid', 'Admin.Notifications.Error'),
+                        ]
+                    ),
+                ],
             ])
             ->add('back_cookie_lifetime', TextType::class, [
-                'required' => true,
+                'label' => $this->trans('Lifetime of back office cookies', 'Admin.Advparameters.Feature'),
+                'help' => $this->trans('When you access your back office and decide to stay logged in, your cookies lifetime defines your browser session. Set here the number of hours during which you want them valid before logging in again.', 'Admin.Advparameters.Help'),
+                'constraints' => [
+                    new Type(
+                        [
+                            'value' => 'numeric',
+                            'message' => $this->trans('The field is invalid', 'Admin.Notifications.Error'),
+                        ]
+                    ),
+                    new GreaterThanOrEqual(
+                        [
+                            'value' => 0,
+                            'message' => $this->trans('The field is invalid', 'Admin.Notifications.Error'),
+                        ]
+                    ),
+                ],
             ])
             ->add('cookie_samesite', ChoiceType::class, [
-                'required' => true,
+                'label' => $this->trans('Cookie SameSite', 'Admin.Advparameters.Feature'),
```

Comment:
Are this wording and the following one new?

Reply:
I think this wording is brand new but my PR is not the one that introduces it. 

This PR setting was introduced this setting(and wording with it I guess)
https://github.com/PrestaShop/PrestaShop/pull/20601/files

Here is the screenshot for where it appears.
![image](https://user-images.githubusercontent.com/47781757/99527273-34f8df00-29a5-11eb-9134-48f6083f80ed.png)

Reply:
Perfect then, thank you very much @JevgenijVisockij!